GHzDACs/ghz_fpga_server.py

```diff
@@ -646,34 +654,43 @@ def run(self, runners, reps, setupPkts, setupState, sync, getTimingData,
                     needSetup = (not setupState) or (not self.setupState) or \
                                     (not (setupState <= self.setupState))
                     if needSetup:
+                        logging.info('needSetup = True')
                         # we require changes to the setup state so first, wait
                         # for triggers indicating that the previous run has
                         # collected.
                         # If this fails, something BAD happened!
                         r = yield waitPkt.send()
+                        logging.info("waitPkt sent")
                         try:
                             # Then set up
                             # print "fpga server: type(setupPkts): ",type(setupPkts)
                             # for idx, sp in enumerate(setupPkts):
-                                # print "type setupPkt[%d]: %s.  setupPkt: " % (idx, type(sp))
-                                # print str(sp)
-                                # for idx, rec in enumerate(sp._packet):
-                                    # print "record %d types: %s, %s, %s, %s" % (idx, type(rec[0]), type(rec[1]), type(rec[2]), type(rec[3]))
-                                    # print "record %d data %s,%s,%s,%s" % (idx, rec[0], rec[1], rec[2], rec[3])
+                            #     print "type setupPkt[%d]: %s.  setupPkt: " % (idx, type(sp))
+                            #     print str(sp)
+                            #     for idx, rec in enumerate(sp._packet):
+                            #         print "record %d types: %s, %s, %s, %s" % (idx, type(rec[0]), type(rec[1]), type(rec[2]), type(rec[3]))
+                            #         print "record %d data %s,%s,%s,%s" % (idx, rec[0], rec[1], rec[2], rec[3])
```

Comment:
I'm not sure what you mean by this. I will remove the commented print statements because we now use the logging module.